class/practice2.2/main.cpp

```diff
@@ -0,0 +1,133 @@
+#include <string>
+#include <vector>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <ranges>
+std::vector<std::string> splitString(const std::string& str, char delimiter) {
+    auto view = str | std::ranges::views::split(delimiter) | std::ranges::views::transform([](auto&& subrange) {
+        return std::string(subrange.begin(), subrange.end());
+        });
+    return std::vector<std::string>(view.begin(), view.end());
+}
+
+std::string listToStr(const std::vector<std::string>& list) {
```

Comment:
There should be a better way something like:
`list | std::views::join_with(" ")`
cause in this case your result will start from " " which may be not what you want